docs/modules/operation/pages/deep-dive/service-assurance/path-outages.adoc

```diff
@@ -17,9 +25,12 @@ If `default-gw-01` is down, it is not possible to reach any node in the two netw
 In this case, an administrator would like to receive just one notification for `default-gw-01`; notifications for the other nodes behind it should not be generated.
 Building this configuration in {page-component-title} requires the following information:
 
-* *Parent Foreign Source:* the foreign source where the parent node is defined.
-* *Parent Foreign ID:* the foreign ID of the parent node this node depends on.
-* *IP Interface:* the primary IP interface is used as the critical IP.
+Parent Foreign Source:: The foreign source where the parent node is defined.
+If the parent node is in the same requisition as the child node, this field can be left blank.
+Parent Foreign ID:: The foreign ID of the parent node this node depends on.
+It is possible to define the parent node based on the label, though using the ID is more reliable in the case the node label changes in the future.
```

Comment:
```suggestion
It is possible to define the parent node based on the label, though using the ID is more reliable in case the node label changes in the future.
```